components/sensor/ld2420.rst

```diff
@@ -189,6 +180,7 @@ Configuration variables:
 ************************
 
 - **operating_mode** (*Required*): Provisions the operating mode select component.
+- All other options from :ref:`Select <config-select>`.
```

Comment:
These are part of the keys like `operating_mode`, not extra config items alongside it
```suggestion
  All other options from :ref:`Select <config-select>`.
```

Reply:
The same applies to all the rest too, each sub-number etc should have this text in each one.

Reply:
Wow, yeah, thank you...wasn't thinking about it in the right way. All fixed now, though. 😄